src/execute.js

```diff
@@ -108,12 +108,16 @@ export function buildRequest({
         value = parameters
       }
 
-      value = parameter && parameter.name && parameters[parameter.name]
+      value = (parameter && parameter.name && parameters[parameter.name]) || value
```

Comment:
The if statement above is not dead code anymore.  I can refactor it a bit to make it more readable.  Parameter could technically be null since it is just iterating through an array though there is code above this line that would throw an error if parameter is null.  Also, that code checking parameter was there before my change so I assumed there is a reason it is there.  I can remove it since it seems pointless